src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,25 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private int temperature;
+    private int volume;
+    private int pressure;
+    private int speed;
+    private int age;
 
+    public Engine(int temperature, int volume, int pressure, int speed, int age) {
+        this.temperature = temperature;
+        this.volume = volume;
+        this.pressure = pressure;
+        this.speed = speed;
+        this.age = age;
+    }
+
+    @Override
+    protected Engine clone() throws CloneNotSupportedException {
```

Comment:
does it really throws that exception or it's has a grey color?

Reply:
What do you mean asking about grey color?

Reply:
![image](https://user-images.githubusercontent.com/50640794/89185166-76a27d00-d5a2-11ea-8fd9-df3f9c34720b.png)

Reply:
remove it if it's grey(which means it's never throws that exceptions) but probably you were using super.clone() and it has been left after that